### PR TITLE
refactor: clean up

### PR DESCRIPTION
**Pull request by epiqueras**

### CODE REVIEW

**Comment by epiqueras** [comment]
Tests are failing because of #121.

File: contracts/standard/arbitration/ArbitrableTokens/LockedToken.sol [hunk 1]
```diff
@@ -55,14 +55,16 @@ contract LockedToken is MintableToken {
      *  @param _to The address to unlock tokens from.
      */
     function partialUnlock(address _to, uint _maxUnlock) public {
-        if (lastUnlock[_to].add(4 weeks) <= now && amountLocked[_to]!=0) {
-            uint amountOfMonths = now.sub(lastUnlock[_to]) / (4 weeks);
+        // solium-disable-next-line security/no-block-members
```
Comment (ferittuncer) [inline contracts/standard/arbitration/ArbitrableTokens/LockedToken.sol]: If we update dependency to the latest version of kleros-scripts we can get rid of these comment lines.

File: contracts/standard/arbitration/ArbitrableDeposit.sol [hunk 2]
```diff
@@ -37,9 +37,9 @@ contract ArbitrableDeposit is Arbitrable {
     uint8 constant CLAIMANT_WINS = 2;
     string constant RULING_OPTIONS = "Owner wins;Claimant wins"; // A plain English of what rulings do. Need to be redefined by the child class.
 
-    modifier onlyOwner{ require(msg.sender == address(owner)); _; }
-    modifier onlyNotOwner{ require(msg.sender != address(owner)); _;}
-    modifier onlyClaimant{ require(msg.sender == address(claimant)); _;}
+    modifier onlyOwner{require(msg.sender == address(owner), "Can only be called by the owner."); _;}
+    modifier onlyNotOwner{require(msg.sender != address(owner), "Cannot be called by the owner."); _;}
+    modifier onlyClaimant{require(msg.sender == address(claimant), "Can only be called by the claimant."); _;}
```
Comment (clesaege) [inline contracts/standard/arbitration/ArbitrableDeposit.sol]: The doc uses space before and after for one line modifier. So let's follow the doc.

File: contracts/standard/arbitration/ArbitrableDeposit.sol [hunk 3]
```diff
@@ -68,14 +74,14 @@ contract ArbitrableDeposit is Arbitrable {
      */
     function deposit() public payable onlyOwner {
         amount += msg.value;
-        address(this).send(msg.value);
+        address(this).transfer(msg.value);
```
Comment (clesaege) [inline contracts/standard/arbitration/ArbitrableDeposit.sol]: Why would we send to itself?

File: contracts/standard/arbitration/ArbitrableDeposit.sol [hunk 4]
```diff
@@ -107,10 +113,12 @@ contract ArbitrableDeposit is Arbitrable {
     function payArbitrationFeeByOwner() public payable onlyOwner{
         uint arbitrationCost = arbitrator.arbitrationCost(arbitratorExtraData);
         ownerFee += msg.value;
-        require(ownerFee == arbitrationCost); // Require that the total pay at least the arbitration cost.
-        require(status < Status.DisputeCreated); // Make sure a dispute has not been created yet.
+        // Require that the total pay at least the arbitration cost.
+        require(ownerFee == arbitrationCost, "Owner fee must equal arbitration cost.");
+        require(status < Status.DisputeCreated, "A dispute has already been raised."); // Make sure a dispute has not been created yet.
 
-        lastInteraction = now;
```
Comment (clesaege) [inline contracts/standard/arbitration/ArbitrableDeposit.sol]: There is nothing wrong with no which is an alias to block.timestamp, now is way more readable.

File: contracts/standard/arbitration/ArbitrableDeposit.sol [hunk 5]
```diff
@@ -126,10 +134,12 @@ contract ArbitrableDeposit is Arbitrable {
     function payArbitrationFeeByClaimant() public payable onlyClaimant {
         uint arbitrationCost = arbitrator.arbitrationCost(arbitratorExtraData);
         claimantFee += msg.value;
-        require(claimantFee == arbitrationCost); // Require that the total pay at least the arbitration cost.
-        require(status<Status.DisputeCreated); // Make sure a dispute has not been created yet.
+        // Require that the total pay at least the arbitration cost.
+        require(claimantFee == arbitrationCost, "Claimant fee must equal arbitration cost.");
+        require(status < Status.DisputeCreated, "A dispute has already been raised."); // Make sure a dispute has not been created yet.
 
-        lastInteraction = now;
```
Comment (clesaege) [inline contracts/standard/arbitration/ArbitrableDeposit.sol]: now is equivalent and more understandable

File: contracts/standard/arbitration/ArbitrableDeposit.sol [hunk 6]
```diff
@@ -150,17 +160,19 @@ contract ArbitrableDeposit is Arbitrable {
     /** @dev Reimburse owner if claimant fails to pay the fee.
      */
     function timeOutByOwner() public onlyOwner {
-        require(status == Status.WaitingClaimant);
-        require(now >= lastInteraction+timeout);
```
Comment (clesaege) [inline contracts/standard/arbitration/ArbitrableDeposit.sol]: now is fine

File: contracts/standard/arbitration/ArbitrableTokens/FundingVault.sol [hunk 7]
```diff
@@ -142,50 +154,54 @@ contract FundingVault is Arbitrable {
...
 
 
-        milestone.feeTeam+=msg.value;
+        milestone.feeTeam += msg.value;
         if (milestone.feeHolders>=arbitrationCost) { // Enough has been paid by all sides.
             createDispute(_milestoneID,arbitrationCost);
         }
         else if (milestone.lastTotalFeePayment==0) { // First time the fee is paid.
-            milestone.lastTotalFeePayment=now;
-            milestone.lastTotalFeePaymentIsTeam=true;
```
Comment (clesaege) [inline contracts/standard/arbitration/ArbitrableTokens/FundingVault.sol]: now is fine

File: contracts/standard/arbitration/ArbitrableDeposit.sol [hunk 8]
```diff
@@ -150,17 +160,19 @@ contract ArbitrableDeposit is Arbitrable {
...
     }
 
     /** @dev Pay claimant if owner fails to pay the fee.
      */
     function timeOutByClaimant() public onlyClaimant {
-        require(status == Status.WaitingOwner);
-        require(now >= lastInteraction+timeout);
+        require(status == Status.WaitingOwner, "The contract is not waiting for the owner.");
+        // solium-disable-next-line security/no-block-members
+        require(block.timestamp >= lastInteraction + timeout, "Not enough time has passed.");
```
Comment (clesaege) [inline contracts/standard/arbitration/ArbitrableDeposit.sol]: now is fine

File: contracts/standard/arbitration/ArbitrableTokens/FundingVault.sol [hunk 9]
```diff
@@ -194,30 +210,38 @@ contract FundingVault is Arbitrable {
...
+        Milestone storage milestone = milestones[_milestoneID];
+        require(msg.sender == team, "Can only be called by the team.");
+        require(!milestone.disputed, "Milestone cannot be disputed.");
+        require(
+            milestone.voteToken.balanceOf(this) < (disputeThreshold*milestone.voteToken.totalSupply()) / 1000,
+            "Not enough votes."
+        ); // There is not enough votes.
+        require(
+            // solium-disable-next-line security/no-block-members, indentation
+            (block.timestamp - milestone.claimTime) > claimToWithdrawTime + (additionalTimeToWithdraw * milestone.voteToken.balanceOf(this)) / (1000 * milestone.voteToken.totalSupply()),
```
Comment (clesaege) [inline contracts/standard/arbitration/ArbitrableTokens/FundingVault.sol]: now is fine

File: contracts/standard/arbitration/CriptoKitties/Auction/ClockAuction.sol [hunk 10]
```diff
@@ -20,11 +20,11 @@ contract ClockAuction is Pausable, ClockAuctionBase {
     /// @param _cut - percent cut the owner takes on each auction, must be
     ///  between 0-10,000.
     function ClockAuction(address _nftAddress, uint256 _cut) public {
-        require(_cut <= 10000);
+        require(_cut <= 10000, "Cut must be less than 10000.");
```
Comment (clesaege) [inline contracts/standard/arbitration/CriptoKitties/Auction/ClockAuction.sol]: Those arz cryptokitty files not our. No need to change anything.

File: contracts/standard/permission/ArbitrablePermissionList.sol [hunk 11]
```diff
@@ -247,16 +257,18 @@ contract ArbitrablePermissionList is PermissionInterface, Arbitrable {
      */
     function executeRequest(bytes32 _value) public {
         Item storage item = items[_value];
-        require(now - item.lastAction >= timeToChallenge);
-        require(!item.disputed);
+        // solium-disable-next-line security/no-block-members
+        require(block.timestamp - item.lastAction >= timeToChallenge, "The time to challenge has not passed yet.");
```
Comment (clesaege) [inline contracts/standard/permission/ArbitrablePermissionList.sol]: now is fine

File: contracts/standard/arbitration/MultipleArbitrableTransaction.sol [hunk 12]
```diff
@@ -124,16 +124,20 @@ contract MultipleArbitrableTransaction {
...
-        require(transaction.status < Status.DisputeCreated); // Make sure a dispute has not been created yet.
-
-        transaction.lastInteraction = now;
-        if (transaction.buyerFee < arbitrationCost) { // The partyB still has to pay. This can also happens if he has paid, but arbitrationCost has increased.
+        // Require that the total pay at least the arbitration cost.
+        require(transaction.sellerFee >= arbitrationCost, "The seller fee must cover arbitration costs.");
+        // Make sure a dispute has not been created yet.
+        require(transaction.status < Status.DisputeCreated, "Dispute has already been created.");
+
+        // solium-disable-next-line security/no-block-members
```
Comment (clesaege) [inline contracts/standard/arbitration/MultipleArbitrableTransaction.sol]: now is fine

File: contracts/standard/arbitration/CriptoKitties/Auction/SaleClockAuction.sol [hunk 13]
```diff
@@ -42,14 +42,15 @@ contract SaleClockAuction is ClockAuction {
...
+        require(msg.sender == address(nonFungibleContract), "The caller must be the non fungible contract.");
         _escrow(_seller, _tokenId);
         Auction memory auction = Auction(
             _seller,
             uint128(_startingPrice),
             uint128(_endingPrice),
             uint64(_duration),
-            uint64(now)
+            // solium-disable-next-line security/no-block-members
+            uint64(block.timestamp)
```
Comment (clesaege) [inline contracts/standard/arbitration/CriptoKitties/Auction/SaleClockAuction.sol]: now is fine

File: contracts/standard/arbitration/ArbitrableDeposit.sol (hunk 2 above)
Comment (epiqueras) [reply contracts/standard/arbitration/ArbitrableDeposit.sol]: Solium doesn't allow changing this. Don't you think it's OK as long as it is consistent?

**Pull request merged**